pkg/liquidity-mining/contracts/gauges/SingleRecipientGaugeFactory.sol

```diff
@@ -44,6 +45,20 @@ contract SingleRecipientLiquidityGaugeFactory {
         return _isGaugeFromFactory[gauge];
     }
 
+    /**
+     * @notice Returns the gauge which sends funds to `recipient`.
+     */
+    function getRecipientGauge(address recipient) external view returns (ILiquidityGauge) {
+        return ILiquidityGauge(_recipientGauge[recipient]);
+    }
+
+    /**
+     * @notice Returns the recipient of `gauge`.
+     */
+    function getGaugeRecipient(address gauge) external view returns (address) {
+        return ISingleRecipientLiquidityGauge(gauge).getRecipient();
+    }
```

Comment:
Is this for lazy callers? :stuck_out_tongue:

Reply:
Yeah, I'm starting to think that assuming anyone looking up info on the gauge system is lazy is a good habit in general.